Review branch week1

white9p opened the pull request:
This files were created to review html/css codes.

white9p committed:
feat: Add header and style

- navigation menu

white9p committed:
feat: fix wrong indentation

-beautify html codes

heejinlee07 commented on cloneGitHubHeader.html:
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Github</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
```
`<Head>`영역에 favicon을 추가하면 좋습니다. 
일반적으로 모든 사이트에서는 favicon이 사용되고 있습니다. 
자세한 사용방법은 아래 링크의 글을 읽어보시기 바랍니다. 
favicon외에도 `<Head>`영역에 필수로 사용되는 태그들에 대한 정보가 있습니다.

[https://developer.mozilla.org/ko/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#맞춤_아이콘_추가하기](https://developer.mozilla.org/ko/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#%EB%A7%9E%EC%B6%A4_%EC%95%84%EC%9D%B4%EC%BD%98_%EC%B6%94%EA%B0%80%ED%95%98%EA%B8%B0)

white9p replied:
favicon을 꼭 추가하겠습니다.
추가완료하였습니다:)

heejinlee07 commented on cloneGitHubHeader.html:
```diff
@@ -0,0 +1,41 @@
+                <div class="logo">
+                    <img src="https://heropcode.github.io/GitHub-Responsive/img/logo.svg" alt="Github logo" />
+                </div>
+                <div class="menu clearfix">
+                    <div class="menu-item">Why GitHub?</div>
+                    <div class="menu-item">Team</div>
+                    <div class="menu-item">Enterprise</div>
+                    <div class="menu-item">Explore</div>
+                    <div class="menu-item">Marketplace</div>
+                    <div class="menu-item">Pricing</div>
+                </div>
```
화면을 구성할 때 시멘틱 태그를 고려하시면서 코드를 작성하는 습관을 들이시는 것이 좋습니다. 따라서 `<div>`로 작성하셔도 되지만 예를들어 `<ul>`요소 안에 `<li>`가 있는 구조로 작성하는 것이 시멘틱하게 코드를 구성하는 것입니다. 관련된 내용은 아래 링크를 참조하세요:)
[https://snusang.tistory.com/4](https://snusang.tistory.com/4)

white9p replied:
시멘틱 태그에 대해 ul li 뿐만아니라 다른것도 더 공부하고 유의해야 하겠습니다.
ul li는 적용 완료하였습니다

heejinlee07 commented on cloneGitHubHeader.html:
```diff
@@ -0,0 +1,41 @@
+    </head>
+</body>
+
+</html>
```
코드 마지막에 엔터를 입력하여 개행을 넣어주세요. 관련된 내용은 아래 링크를 보시면 됩니다.
https://minz.dev/19

white9p replied:
개행완료하였습니다

heejinlee07 commented on cloneGitHubHeader.html:
```diff
@@ -0,0 +1,41 @@
+        <div class="container clearfix">
+            <div class="container-left clearfix">
+                <div class="logo">
+                    <img src="https://heropcode.github.io/GitHub-Responsive/img/logo.svg" alt="Github logo" />
```
`<img>` 태그에 대체 텍스트를 넣으신 점이 좋아요. 항상 SEO 최적화를 위해 이미지 태그에는 대체 텍스트를 작성하는 습관을 들이세요:)

white9p replied:
감사드립니다:)

heejinlee07 commented on main.css:
```diff
@@ -0,0 +1,54 @@
+    margin:0;
+    padding:0;
+}
+.container{
+width:980px;
+margin:auto;
+}
```
코드 인덴테이션이 맞지 않네요. 다른 코드와 동일한 간격으로 작성해주세요!

white9p replied:
에고.. 자동 인덴트 설정 추가 하고 코드도 수정하였습니다.

heejinlee07 commented on main.css:
```diff
@@ -0,0 +1,54 @@
+width:980px;
+margin:auto;
+}
+.container-left{
+   float:left;
+   padding:20px 0;
+}
+.logo{
+    float:left;
+    padding-right:5px;
+}
+.logo img{
+    display:block;
+    
+}
+.menu{
+    float:left;
+    
+}
+.menu-item{
+    float:left;
+    padding:8px 10px;
+}
```
float는 알아 두어야 하지만, 현업에서 거의 쓰지 않는 속성입니다. 
float를 쓰지 않는 이유는 기존의 요소의 정렬을 깨트리고, 
잘못 사용할 시 여러 사이드 이펙트를 일으킬 가능성이 높기 때문입니다. 
무엇보다도 float를 대체할 수 있는 flexbox와 grid가 등장했기 때문에 float는 쓰지 않습니다. 
flexbox를 통해 화면을 구성하는 방법을 공부해보시면 좋겠습니다.

관련하여 읽어보면 좋은 링크는 아래와 같습니다.

https://blog.eunsatio.io/develop/%EB%A0%88%EC%9D%B4%EC%95%84%EC%9B%83-%EC%A0%95%EB%A0%AC%EC%9D%84-%EC%9C%84%ED%95%B4-css-float%EB%A5%BC-%EB%90%98%EB%8F%84%EB%A1%9D-%EC%93%B0%EC%A7%80-%EC%95%8A%EC%95%84%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0

flexbox를 쉽고 재밌게 공부할 수 있는 사이트입니다.
https://flexboxfroggy.com/

heejinlee07 commented on main.css:
```diff
@@ -0,0 +1,54 @@
+    border:1px solid black;
+    border-radius: 6px;
+    margin:4px 0 4px 10px;
+}
```
엔터를 입력하여 개행을 넣어주세요!

white9p replied:
개행 수정하였습니다:)

heejinlee07 reviewed (commented):
안녕하세요 반갑습니다 😃
첫 리뷰를 작성하시느라 고생많으셨습니다.

float로 레이아웃 배치를 잘해주셨습니다. 
하지만 현업에서는 flexbox, grid가 주로 사용되기 때문에 
이에 대한 연습을 하시면 도움이 될 거예요!

그리고, 현업에서는 보통 Conversation 란에 개발한 내용에 대해 기록하여 pr을 올리기 때문에
앞으로 pr 리퀘스트 요청 시 Conversation 란에 자신이 작업한 내용을 상세히 작성해주시면 좋습니다:)

white9p committed:
feat: changed minor mistakes on main.css

-inserted final newline.
-fix unindented lines on .container

white9p committed:
feat: inserted faviocon link  on head and final newline at the end of file  on cloneGitHubHeader.html

white9p committed:
feat: used semantic tag on menu on head on cloneGitHubHeader.html

chanaged div tags into  ul and li tag

white9p committed:
feat: changed codes on main.css to match the prior changes on cloneGitHubHeader.html

-according to tag changes which was div into ul and li , inserted list style and margin changes on menu class on main.css

white9p commented:
- pr의 conversation란에 개발한 내용을 적는 습관을 들이겠습니다 👍
